$NetBSD: patch-src_plugins_ruby_CMakeLists.txt,v 1.2 2023/07/10 08:16:59 jperkin Exp $

RUBY_LDFLAGS may contain strings unsuitable for target_link_libraries(), for
example "-arch arm64" will get converted to "-arch -larm64".

--- src/plugins/ruby/CMakeLists.txt.orig	2023-06-30 19:18:01.000000000 +0000
+++ src/plugins/ruby/CMakeLists.txt
@@ -29,7 +29,7 @@ if(RUBY_FOUND)
   # see: https://github.com/ruby/ruby/pull/7085
   add_definitions(-Wno-unused-parameter)
   include_directories(${RUBY_INCLUDE_DIRS})
-  target_link_libraries(ruby ${RUBY_LDFLAGS} weechat_plugins_scripts coverage_config)
+  target_link_libraries(ruby ${RUBY_LIBRARIES} weechat_plugins_scripts coverage_config)
 endif(RUBY_FOUND)
 
 install(TARGETS ruby LIBRARY DESTINATION "${WEECHAT_LIBDIR}/plugins")
